knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -291,6 +293,26 @@ func (r *ReconcileKnativeKafka) transform(manifest *mf.Manifest, instance *serve
 	return nil
 }
 
+func replaceJobGenerateName() mf.Transformer {
+	version := os.Getenv("CURRENT_VERSION")
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() == "Job" {
+			job := &batchv1.Job{}
+			if err := scheme.Scheme.Convert(u, job, nil); err != nil {
+				return err
+			}
+			if job.GetName() == "" && job.GetGenerateName() != "" {
+				job.SetName(fmt.Sprintf("%s%s", job.GetGenerateName(), version))
+				job.SetGenerateName("")
```

Comment:
this is a bit different than upstreams. `jobs.go` transformer

should we also align that upstream?